android/tests_backend/widgets/label.py

```diff
@@ -31,3 +31,7 @@ def alignment(self):
             None if Build.VERSION.SDK_INT < 26 else self.native.getJustificationMode()
         )
         return toga_alignment(self.native.getGravity(), justification_mode)
+
+    @property
+    def vertical_alignment(self):
+        return toga_vertical_alignment(self.native.getGravity())
```

Comment:
This will have to be implemented by the other backends.

Reply:
I've done this for iOS, Cocoa and GTK; although for iOS and Cocoa, it's a bit of a non-test because there's literally no property to inspect to verify vertical alignment - it's just what the widget does (or, in the case of iOS Label, what the widget has been overridden to do).